docs/xxjitservershareromclasses.md

```diff
@@ -23,18 +23,18 @@
 
 # -XX:\[+|-\]JITServerShareROMClasses
 
-This option enables the JITServer server to share cached ROM classes between JITServer clients.
+This option enables or disables the JITServer server to share cached ROM classes between JITServer clients.
 
 ## Syntax
 
         -XX:[+|-]JITServerShareROMClasses
 
 | Setting                 | Effect | Default                                                                            |
 |-------------------------|--------|:----------------------------------------------------------------------------------:|
-|`-XX:+JITServerShareROMClasses`           | Enable |                                                                                    |
-|`-XX:-JITServerShareROMClasses`           | Disable| :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+|`-XX:+JITServerShareROMClasses`           | Enable |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>  |
+|`-XX:-JITServerShareROMClasses`           | Disable|                                                                                        |
 
- :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** The `-XX:+JITServerShareROMClasses` option is enabled by default at the server on specifying the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) option.
+ :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** The `-XX:+JITServerShareROMClasses` option is enabled by default at the server because of the default setting of the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) option. If you've disabled the JITServer AOT cache at the server (`-XX:-JITServerUseAOTCache`), specify the `-XX:+JITServerShareROMClasses` option explicitly to enable this feature.
```

Comment:
Suggestion:
"**Note:** The `-XX:+JITServerShareROMClasses` option is enabled by default at the server because it is a per-requisite for the JITServer AOT caching feature which is now enabled by default. If you've disabled the JITServer AOT cache at the server (`-XX:-JITServerUseAOTCache`), specify the `-XX:+JITServerShareROMClasses` option explicitly to enable this feature."